test/reference_directory_content.json

```diff
@@ -1,236 +1,311 @@
-    "1000genome/bcf/1000genome_chr15.bcf": 458839209,
-    "1000genome/bcf/1000genome_chr9.bcf.csi": 100696,
-    "1000genome/bcf/1000genome_chr8.bcf.csi": 120454,
-    "1000genome/bcf/1000genome_chr10.bcf.csi": 111843,
-    "1000genome/bcf/1000genome_chr5.bcf.csi": 148879,
-    "1000genome/bcf/1000genome_chr1.bcf.csi": 182048,
-    "1000genome/bcf/1000genome_chr13.bcf": 559093113,
-    "1000genome/bcf/1000genome_chr7.bcf": 910616163,
-    "1000genome/bcf/1000genome_chr5.bcf": 991043474,
-    "1000genome/bcf/1000genome_chr16.bcf": 495726157,
-    "1000genome/bcf/1000genome_chr2.bcf.csi": 200334,
-    "1000genome/bcf/1000genome_chr3.bcf": 1108026434,
-    "1000genome/bcf/1000genome_chr15.bcf.csi": 66132,
-    "1000genome/bcf/1000genome_chr13.bcf.csi": 79552,
-    "1000genome/bcf/1000genome_chr9.bcf": 674181206,
-    "1000genome/bcf/1000genome_chr1.bcf": 1218805649,
-    "1000genome/bcf/1000genome_chr14.bcf": 507978610,
-    "1000genome/bcf/1000genome_chr18.bcf.csi": 62374,
-    "1000genome/bcf/1000genome_chr12.bcf": 743123551,
-    "1000genome/bcf/1000genome_chr8.bcf": 862933354,
-    "1000genome/bcf/1000genome_chr20.bcf.csi": 49191,
-    "1000genome/bcf/1000genome_chr19.bcf": 361013775,
-    "1000genome/affymetrix_chip/all.vcf.gz": 782884333,
-    "1000genome/affymetrix_chip/all.vcf.gz.tbi": 2038827,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr5.txt": 7303158,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr12.txt": 5710169,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr16.txt": 3167749,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr7.txt": 6219422,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr6.txt": 7918268,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr22.txt": 1540819,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr8.txt": 6416103,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr13.txt": 4687436,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr19.txt": 1593309,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chrX_par1.txt": 50520,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr18.txt": 3488316,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr1.txt": 8806085,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr4.txt": 7175980,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr20.txt": 2806690,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr2.txt": 9792987,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr3.txt": 7589745,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr21.txt": 1517797,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr10.txt": 6251004,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr14.txt": 3688036,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr15.txt": 3196892,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr17.txt": 2538845,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chrX_par2.txt": 6854,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr11.txt": 5829848,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chr9.txt": 5297717,
-    "genetic_map_GRCh37/genetic_map_GRCh37_chrX.txt": 3023467
+  "stat_file.txt": 36010,
```

Comment:
What is the motivation behind the using 2 spaces in JSON instead of 4?

Reply:
I formatted it with my IDE, and turns out it does 2 spaces by default. If it bothers you, i can make it 4.

Reply:
Oh, the problem with huge diff still persists... Ok, let's leave it as is.